docs/apidocs/simulation-quimb.rst

```diff
@@ -0,0 +1,27 @@
+##########################################################################
+Quimb simulation backend (:mod:`qiskit_addon_aqc_tensor.simulation.quimb`)
+##########################################################################
+
+.. automodule:: qiskit_addon_aqc_tensor.simulation.quimb
+   :no-members:
+   :no-inherited-members:
+   :no-special-members:
+
+.. autosummary::
+    :toctree: ../stubs/
+    :nosignatures:
+
+    QuimbCircuitFactory
+    QuimbSimulator
+    QiskitQuimbConversionContext
+
+Functions
+=========
```

Comment:
Is there a better header name or set of headers we can use?

Reply:
I can't think of one at the moment.  But if I had to bisect them somehow, I would put `is_*_available()` functions is one bucket and everything else in another.  The quimb ones are really for advanced users who want to call into quimb directly.  The basic workflow doesn't require a user to use these functions directly, but rather they are used under the hood).